### PR TITLE
Sidepieces

### PR DESCRIPTION
**Pull request by therrick47**

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "type": "chrome",
+            "request": "launch",
+            "name": "Launch Chrome against localhost",
+            "url": "http://localhost:8080",
+            "webRoot": "${workspaceFolder}"
+        }
+    ]
```
Comment (savvymatt) [inline .vscode/launch.json:14]: This is so C# of you. 🤣

You don't just run npm start like a normie?

**Review by savvymatt — COMMENTED** [review]
Code looks pretty damn good to me, I'm sure there will be things to abstract and changes to be made to how it all works as you move along, but not bad at all for a first touch with React and JS

File: .vscode/launch.json (hunk 1 above)
Comment (therrick47) [reply .vscode/launch.json:14]: I did end up doing that. Was messing around VSCode launch stuff. Probably a good idea for me to clean this up in the future haha.

Comment (savvymatt) [reply .vscode/launch.json:14]: 🤣🤣🤣 all good if you prefer it this way, it's just funny

**Pull request merged**